cfpb/cfgov-refresh
# Remove legacy nemo assets

I don't see how these assets are used, but please scrutinize. It's possible they are used in a way I'm not seeing.

## Removals

- Removes unused nemo legacy assets.
- Removes unused nemo legacy gulp tasks and configuration.
- Removes eot fonts and unused font duplicates.

## How to test this PR

1. Pull branch and build and site should work as before. 
2. I don't have a specific area to test as I don't see how these are used, other than pages like /your-story/ that still reference nemo paths and should be updated some time.
Also, /jobs/supervision/, /fair-lending/, and /practitioner-resources/students/helping-borrowers-find-ways-to-stay-afloat/
